zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -856,6 +856,10 @@ property, when available, is noted below.
     commit log is triggered.
     Does not affect the limit defined by *flushDelay*.
     Default is 1000.
+* *enforeQuota* :
+    (Java system property: **zookeeper.enforeQuota**)
+    enfore the quota check. Enable this option to use the [quota feature](http://zookeeper.apache.org/doc/current/zookeeperQuotas.html).
+    the default value:true.
```

Comment:
this should be disabled by default, otherwise applications may break

Reply:
Actually, the original soft quota limit(print warning log) had already enabled by default.
Turning on `enforeQuota` will not break the compatibility

Reply:
yes it breaks because before this change in case of quota related error you had only a log line, now the client will be blocked